altair/_magics.py

```diff
@@ -76,12 +82,12 @@ def _get_variable(name):
     nargs="*",
     help="local variable name of a pandas DataFrame to be used as the dataset",
 )
-@magic_arguments.argument("-v", "--version", dest="version", default="5")
+@magic_arguments.argument("-v", "--version", dest="version", default="v5")
```

Comment:
I made this change and the one below so that this will be easier to detect in the future. It was giving me issues initially and it is hard to grep the repo for just the version number without the `v` prefix.